docs.md

```diff
@@ -0,0 +1,54 @@
+# [<](README.md) &nbsp; Conventions Documentation Guidelines
+
+## Rebuilding the content table
+
+Actions which requires the main content table to be updated:
+
+1. Adding a file
+1. Moving a file
+1. Deleting a file
+1. Updating a file's main header
+
+To update the content table:
+
+1. You would need to install the dependencies by running:
+
+    ```sh
+    yarn
+    ```
+
+2. To rebuild, just run:
+
+    ```sh
+    yarn generate-toc
+    ```
+
+## Adding a document
+
+When adding a  document file for this repo it should follow this structure:
+
+```markdown
+
+# [<](README.md) &nbsp; ${main_header}
```

Comment:
It doesn't seem like the back-to-top links work unless you format the main_header_dashed like so `--conventions-documentation-guidelines`. This is because of the `" &nbsp; "` formatting.

How about:

```md
# ${main_header}
[<- BACK](README.md)

${table_of_content}

&nbsp;

${content}

[Back to the top](#${main_header_dashed})
```

Doing away with the back in the header and just making it its own line?

Reply:
I really like it in the same line so I'm just adding `--` everywhere